### DISCUSSION TITLE
posts/2023/11/riding-trend-my-defaults/

### DISCUSSION THREAD

**Discussion by giscus** [brycewray/comments#39] [Announcements]
# posts/2023/11/riding-trend-my-defaults/

Okay, sure, I’ll play along with all you other bloggers.

https://www.brycewray.com/posts/2023/11/riding-trend-my-defaults/

**Comment by andrewcanion** [comment]
I feel honoured that you have participated in something that my podcast kicked off. How the world turns!

**Reply by brycewray** [reply]
@andrewcanion I am embarrassed that I only now have **learned** about your podcast, sir! I have become one of your latest subscribers.

**Comment by andrewcanion** [comment]
Clearly I need to improve my skills of self-promotion. :-)
